src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +52,23 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                    (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+        AuthenticationService authenticationService =
+                    (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        User user = new User();
+        user.setEmail("LikeMyWorld@gmail.com");
+        user.setPassword("123QWE");
+
+        authenticationService.register(user.getEmail(), user.getPassword());
+        System.out.println(authenticationService.login(user.getEmail(), user.getPassword()));
+        System.out.println(user.getEmail());
+        System.out.println(user.getPassword());
```

Comment:
Avoid printing sensitive information like passwords. This is a security concern and should be removed or replaced with a more secure logging mechanism.